### PR TITLE
Add emoji unicode to special keys

### PR DESCRIPTION
**Pull request by Ad1B3nAr0ya**

### CODE REVIEW

File: src/enums/SpecialKeys.ts [hunk 1]
```diff
@@ -10,4 +10,6 @@ export enum SpecialKeys {
   ARROW_DOWN = '\uE015',
   TAB = '\uE004',
   ESCAPE = '\uE00C',
+  EMOJI_HEART = '❤', //In The BMP (basic multilingual plane)
```
Comment (AlexeyCL) [inline src/enums/SpecialKeys.ts]: please check, that ci machine(linux) can read this emojies

File: test/specs/SendKeysSpec.ts [hunk 2]
```diff
@@ -43,4 +43,13 @@ describeCommon('Send Keys', () => {
...
     assert.equal($(TEXT_INPUT).getValue(), `${text}\n\n\n${text}`);
   });
+
+  it('Special Key Emoji', () => {
+    $(TEXT_INPUT).clearValue();
+    $(TEXT_INPUT).click();
+    BrowserUtils.sendKeys(SpecialKeys.EMOJI_HEART);
+    BrowserUtils.sendKeys(SpecialKeys.ENTER);
+    BrowserUtils.sendKeys(SpecialKeys.EMOJI_HEART);
+    assert.equal($(TEXT_INPUT).getValue(), `❤\n❤`);
```
Comment (AlexeyCL) [inline test/specs/SendKeysSpec.ts]: not sure, that it will works in linux machine and almost sure, that in windows it will fail

Comment (Ad1B3nAr0ya) [reply test/specs/SendKeysSpec.ts]: changed to unicode

File: src/enums/SpecialKeys.ts (hunk 1 above)
Comment (Ad1B3nAr0ya) [reply src/enums/SpecialKeys.ts]: fixed and changed to unicode

File: test/specs/SendKeysSpec.ts [hunk 3]
```diff
@@ -43,4 +43,13 @@ describeCommon('Send Keys', () => {
...
     assert.equal($(TEXT_INPUT).getValue(), `${text}\n\n\n${text}`);
   });
+
+  it('Special Key Emoji', () => {
+    $(TEXT_INPUT).clearValue();
+    $(TEXT_INPUT).click();
+    BrowserUtils.sendKeys(SpecialKeys.EMOJI_HEART);
+    BrowserUtils.sendKeys(SpecialKeys.ENTER);
+    BrowserUtils.sendKeys(SpecialKeys.EMOJI_HEART);
+    assert.equal($(TEXT_INPUT).getValue(), `\u2766\n\u2766`);
```
Comment (AlexeyCL) [inline test/specs/SendKeysSpec.ts]: instead of string you can use enum values:
SpecialKeys.EMOJI_HEART

Comment (Ad1B3nAr0ya) [reply test/specs/SendKeysSpec.ts]: done

**Pull request merged**